src/module/item/physical/coins.ts

```diff
@@ -91,7 +91,16 @@ class CoinsPF2e implements Coins {
     /** Parses a price string such as "5 gp" and returns a new CoinsPF2e object */
     static fromString(coinString: string, quantity = 1): CoinsPF2e {
         // This requires preprocessing, as large gold values contain , for their value
-        const priceTag = String(coinString).trim().replace(/,/g, "");
+        const priceTag = DENOMINATIONS.reduce(
+            (s, denomination) => {
+                const localizedDenomination = game.i18n.localize(`PF2E.CurrencyAbbreviations.${denomination}`);
+                if (localizedDenomination === denomination) return s;
+                // This matches localized denomination if it's not followed or preceeded by an unicode letter character
+                const pattern = new RegExp(`(?!<\\p{L})${localizedDenomination}(?!\\p{L})`, "u");
+                return s.replace(pattern, denomination);
+            },
+            String(coinString).trim().replace(/,/g, ""),
```

Comment:
```suggestion
            coinString.trim().replace(/,/g, ""),
```
Not sure why that was getting run through `String` before, but let's do without.

Reply:
Removing `String` seems to be safe, as the existing callers handle the type.
I've tested it and everything looks alright.